### PR TITLE
shadowsocks-v2ray-plugin: Add version 1.3.1

### PR DESCRIPTION
**Pull request by g4li**
The very useful [v2ray plugin for shadowsocks](https://github.com/shadowsocks/v2ray-plugin).

Manually tested.

### CODE REVIEW

File: bucket/shadowsocks-v2ray-plugin.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            "64bit": {
+                "url": "https://github.com/shadowsocks/v2ray-plugin/releases/download/v$version/v2ray-plugin-windows-amd64-v$version.tar.gz"
+            },
+            "32bit": {
+                "url": "https://github.com/shadowsocks/v2ray-plugin/releases/download/v$version/v2ray-plugin-windows-386-v$version.tar.gz"
+            }
+        },
+        "hash": {
+            "url": "https://github.com/shadowsocks/v2ray-plugin/releases/tag/v$version",
+            "regex": "([.*])[\\s]+$basename"
```
Comment (Ash258) [inline bucket/shadowsocks-v2ray-plugin.json:35]: Go with something liek this:

```json
            "regex": "$sha1\\s+bin/$basename"
```

File: bucket/shadowsocks-v2ray-plugin.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            "bin": [
+                [ "v2ray-plugin_windows_amd64.exe", "v2ray-plugin"]
+            ]
+        },
+        "32bit": {
+            "url": "https://github.com/shadowsocks/v2ray-plugin/releases/download/v1.3.1/v2ray-plugin-windows-386-v1.3.1.tar.gz",
+            "hash": "sha1:fcb25cae77b392178efb91c6fb2450fb3f9e7da3",
+            "bin": [
+                ["v2ray-plugin_windows_386.exe", "v2ray-plugin"]
+            ]
```
Comment (Ash258) [inline bucket/shadowsocks-v2ray-plugin.json:20]: Completely nonsense name.

```suggestion
            "bin": [
                [
                    "v2ray-plugin_windows_386.exe",
                    "shadowsocks-v2ray-plugin"
                ]
            ]
```

File: bucket/shadowsocks-v2ray-plugin.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "homepage": "https://github.com/shadowsocks/v2ray-plugin",
+    "license": "MIT",
+    "depends": "shadowsocks",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/shadowsocks/v2ray-plugin/releases/download/v1.3.1/v2ray-plugin-windows-amd64-v1.3.1.tar.gz",
+            "hash": "sha1:8b7ebfbb98a26f5a9cb206a631bda17fcd418bd6",
+            "bin": [
+                [ "v2ray-plugin_windows_amd64.exe", "v2ray-plugin"]
+            ]
```
Comment (Ash258) [inline bucket/shadowsocks-v2ray-plugin.json:13]:
```suggestion
            "bin": [
                [
                    "v2ray-plugin_windows_amd64.exe",
                    "shadowsocks-v2ray-plugin"
                ]
            ]
```

File: bucket/shadowsocks-v2ray-plugin.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "1.3.1",
+    "description": "Yet another SIP003 plugin for shadowsocks, based on v2ray",
```
Comment (Ash258) [inline bucket/shadowsocks-v2ray-plugin.json:3]:
```suggestion
    "description": "SIP003 plugin based on v2ray for shadowsocks",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — COMMENTED** [review]
Manifest is OK. But i cannot verify functionality

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**